docs/index.md

```diff
@@ -207,7 +223,27 @@ async def app(scope, receive, send):
     ...
 
 if __name__ == "__main__":
-    uvicorn.run("example:app", host="127.0.0.1", port=5000, log_level="info")
+    config = uvicorn.Config("main:app", port=5000, log_level="info")
+    server = uvicorn.Server(config)
+    server.run()
+```
+
+If you'd like to run Uvicorn from an already running async environment, use `uvicorn.Server.serve()` instead:
+
+```python
```

Comment:
```suggestion
main.py
```python
```

Reply:
just to say this is the name of the file, I'll approve it without this change but we have too many people not understanding how the app string works

Reply:
Added on the first file here.